test/extended/operators/olm.go

```diff
@@ -70,4 +71,19 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			}
 		})
 	}
+
+	// OCP-24061 - [bz 1685230] OLM operator should use imagePullPolicy: IfNotPresent
+	// author: bandrade@redhat.com
+	g.It("have imagePullPolicy:IfNotPresent on thier deployments", func() {
+		var deploymentResource = [3]string{"catalog-operator", "olm-operator", "packageserver"}
+		for _, v := range deploymentResource {
+			msg, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("-n", "openshift-operator-lifecycle-manager", "deployment", v, "-o=jsonpath={.spec.template.spec.containers[*].imagePullPolicy}").Output()
```

Comment:
nit: is there a `.WithNamespace("<namespace>")` method you can use instead?

Reply:
No, this is not available unfortunately